pkg/koordlet/util/meminfo.go

```diff
@@ -91,7 +91,7 @@ func (i *MemInfo) MemUsageBytes() uint64 {
 
 // MemWithPageCacheUsageBytes returns the usage of mem with page cache bytes.
 func (i *MemInfo) MemUsageWithPageCache() uint64 {
-	// total - available
+	// total - available + activefile + inactivefile
```

Comment:
Why not use `Total-Free`? Could you please add some elaboration?

Reply:
total = used + free + buff/cache, available is complex metric and memAvailable <= MemFree + Active(file) + Inactive(file) + SReclaimable. I think total-free expression is simple and clear.